### PR TITLE
Release 0.1.0

### PR DESCRIPTION
**Pull request by Bachibouzouk**
@rl-institut/wam-team  - After this branch is merged into master we should start enforcing the [Driessen](https://nvie.com/posts/a-successful-git-branching-model/) workflow and keep off master (it will be easier as @christian-rli has setup `dev` as the default branch, so inattention mistakes will not result in a merge into `master` :))

### CODE REVIEW

**Comment by nesnoj** [comment]
Hey, if you want me to review, you may want to add me to the reviewers? ;)

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,6 @@ __pycache__/
 *.pyc
 .pycharmrc
 celery.sh
-
+WAM_APP_*/
```
Comment (nesnoj) [inline .gitignore:7]: What's this for?

**Review by nesnoj — COMMENTED** [review]
Did you check if the installation according to the docs works out?

File: .gitignore (hunk 1 above)
Comment (Bachibouzouk) [reply .gitignore:7]: To leave out the WAM apps out of the version control

**Review by nesnoj — APPROVED** [review]

**Comment by Bachibouzouk** [comment]
> Did you check if the installation according to the docs works out?

@nesnoj - yes I did it one more time on my personal laptop :)

**Comment by christian-rli** [comment]
> Hey, if you want me to review, you may want to add me to the reviewers? ;)

Whoops. Sorry @nesnoj . I added you as reviewer again by mistake. I did also add you as a code owner. That role is not necessary to review a merge to master (we could consider making it so), but it seemed appropriate nonetheless.

**Pull request merged**